docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+# Use postgres/example user/password credentials
+version: "3.9"
+services:
+  esphome:
+    image: esphome/esphome:2024.4
+    volumes:
+      - ./esp32/config:/config
+      - /etc/localtime:/etc/localtime:ro
+      - /dev/ttyUSB0:/dev/ttyUSB0
+    privileged: true
+
+  db:
+    image: postgres:16.3-alpine
+    shm_size: 128mb
+    volumes:
+      - "./database/data:/var/lib/postgresql/data"
+      - "./database/db.sql:/docker-entrypoint-initdb.d/db.sql"
```

Comment:
this is outdated I believe.
It gives:
```
/usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/db.sql
psql:/docker-entrypoint-initdb.d/db.sql: error: could not read from input file: Is a directory
```

Reply:
yes, the docker-file.yml was the development one and is currently outdated, you should use the docker-file.prod.yml for production